conda/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-idl
+  version: {{ data.get('version') }}_{{ TS_XML_VERSION }}
+
+source:
+  git_url: https://github.com/lsst-ts/ts_idl
+  git_rev: v1.1.0
```

Comment:
I think this is forcing the version to be `v1.1.0`. You probably want to change it to 

```
git_rev: {{ idl_version }}
```

and make sure you inject the environment variable when you run the container (see command above).

Reply:
Fixed in commit a27266d0ca07c4a24761d164fcef795645fe1aa4.